### DISCUSSION TITLE
What are the F1~F7 channels below the 200th channel memory?

### DISCUSSION THREAD

**Discussion by MicaelJarniac** [egzumer/uvk5-chirp-driver#39] [Q&A] [answered]
![image](https://github.com/egzumer/uvk5-chirp-driver/assets/19514231/813444f8-a08a-4973-a732-73e888961f4b)

- F1(18M-108M)A
- F1(18M-108M)B
- ...
- F7(470M-1300M)B

**Comment by egzumer** [comment, marked as answer]
Frequency mode current bands settings.

**Reply by MicaelJarniac** [reply]
Is it safe to freely edit them?

**Reply by egzumer** [reply]
Yes
